docs/charts/mermaid/MongoError.mmd

```diff
@@ -0,0 +1,8 @@
+graph TD
```

Comment:
Can you make a small meta readme in this diretory of how to use these files? Nothing major, links to the right places, commands you run to generate the svgs etc.